### ISSUE TITLE
DNSDist: Error message is useless userdata

### ISSUE THREAD

**Issue by kaosdrachen** [PowerDNS/pdns#6535]
- Program:  dnsdist 
 - Issue type: Bug report

### Short description

While scripting an advanced config I encountered a bug in the way dnsdist returns errors 
back to lua that renders the error message as returned by pcall() into a useless userdata object

### Environment
 - Operating system: Debian
 - Software version: 1.3.0
 - Software source: PDNS repository for DNSdist 1.3

### Steps to reproduce
1. Install dnsdist
2. Add the following into a config file:

```
x = newNMG()
print(pcall(function() x:addMask('banana') end))
```
3. Run the code; 

### Expected behaviour
A readable error message; In this case it should return 'Unable to convert 'banana' to a netmask'

### Actual behaviour
it will print 'false userdata' followed by a pointer.

### Other information
I've discussed this in IRC with Habbie; they're the one who recommended filing a bug report.

**Comment by Habbie** [comment]
> it will print 'false userdata' followed by a pointer.

To be clear, the return value from pcall is `false` and the useless `userdata`. This needs a sprinkling of `rethrow_if_nested` and/or some LuaWrapper bindings for exception objects.

**Comment by chbruyand** [comment]
Closing the issue as it has been fixed by #6541
